stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/viewholder/decorator/BackgroundDecorator.kt

```diff
@@ -104,6 +129,9 @@ internal class BackgroundDecorator : BaseDecorator() {
 
     companion object {
         private val MESSAGE_DELETED_BACKGROUND = R.color.stream_grey_light_opacity_50
-        private const val DEFAULT_CORNER_RADIUS = 16f
+        private val MESSAGE_OTHER_STROKE_COLOR = R.color.stream_border_stroke
+        private val MESSAGE_CURRENT_USER_BACKGROUND = R.color.stream_grey_90
+        private const val DEFAULT_CORNER_RADIUS_DP = 16
+        private const val DEFAULT_STROKE_WIDTH_DP = 1
```

Comment:
This stroke width (combined with how light the line is) is barely visible for me in messages like this:

![image](https://user-images.githubusercontent.com/12054216/100878447-7e1f5780-34aa-11eb-8e5a-42d80bcdd3ce.png)

Reply:
Just need to add something like (stoke / 2) margins to the drawable

Reply:
I decided to draw to rectangles. Outer with stroke color, inner with a default background color which smaller by stroke width

Reply:
I didn't have time to put in an actual review on this PR again, but I checked it out now, and it looks perfect with the changes 👍